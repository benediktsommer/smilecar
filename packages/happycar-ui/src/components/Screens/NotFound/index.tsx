import { memo } from 'react';

export default memo(() => <p>NOPE</p>);
