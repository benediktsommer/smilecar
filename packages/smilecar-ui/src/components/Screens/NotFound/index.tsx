import { memo } from 'react';

export default memo(() => <p>There is nothing to see</p>);
