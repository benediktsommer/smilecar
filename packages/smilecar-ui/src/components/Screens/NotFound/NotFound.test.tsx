import { render } from '@testing-library/react';

import NotFound from './';

describe('Test <NotFound />', () => {
  it('should render an html markup', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
