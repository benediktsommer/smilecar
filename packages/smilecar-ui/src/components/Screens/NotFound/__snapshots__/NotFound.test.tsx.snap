// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <NotFound /> should render an html markup 1`] = `
<DocumentFragment>
  <p>
    There is nothing to see
  </p>
</DocumentFragment>
`;
